### PR TITLE
iOS dev env setup

### PR DESCRIPTION
**Pull request by ghost**
So far we have used this branch to setup the development environment, fix build issues, and fix some of the issues that showed up afterwards during the runtime (i.e. the issues that have been closed so far). 

So this pull request includes all the changes so far. 

From now on I'll create a single branch per issue, and will pull request it into 'develop' directly when done.

### CODE REVIEW

File: I Bike CPH/I Bike CPH/Info.plist [hunk 1]
```diff
@@ -62,10 +62,14 @@
...
 	</array>
+	<key>NSLocationWhenInUseUsageDescription</key>
+	<string>This app uses location services to provide route suggestions.</string>
 	<key>UISupportedInterfaceOrientations</key>
 	<array>
 		<string>UIInterfaceOrientationPortrait</string>
 		<string>UIInterfaceOrientationPortraitUpsideDown</string>
 	</array>
+	<key>NSLocationAlwaysAndWhenInUseUsageDescription</key>
+	<string>This app uses location services to provide route suggestions.</string>
```
Comment (emiltin) [inline I Bike CPH/I Bike CPH/Info.plist]: needs i18n (translation to danish also)

**Review by emiltin — CHANGES_REQUESTED** [review]
there's a string that needs translation, efter it's good to merge

**Comment by emiltin** [comment]
ready to merge?
travis is failing, but we can fix that later i think.

**Comment by babak-feiz** [comment]
I think so. It seems that it has to do with the configuration on Travis. It could also be that we need to do some updates on Travis.

**Pull request merged**